include/functions.php

```diff
@@ -73,3 +113,25 @@ function findStartDateOfNextPeriod ($lastStartDate, $cycleLengthInDays) {
     $nextPeriodStartDate = date_add($lastStartDate, $cycleLengthInDays);
     return $nextPeriodStartDate;
 }
+
+
+function getDateToStoreInSession () {
+    if (!isset($_SESSION['date']) ||  isset($_POST['now'])){
+        $monthDisplayed = new DateTime();
+        return $monthDisplayed;
+    }
+    
+    else if (isset ($_POST['prev'])) {
+        $monthInterval = new DateInterval("P1M");
+        $monthDisplayed = date_sub($_SESSION['date'], $monthInterval);
+        return $monthDisplayed;
+
+    }
+    
+    else if (isset ($_POST['next'])) {
+        $monthInterval = new DateInterval("P1M");
+        $monthDisplayed = date_add($_SESSION['date'], $monthInterval);
+        return $monthDisplayed;
```

Comment:
You could remove some duplicate code here by returning $monthDisplayed at the end of the chain of if/else statements